src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActiveState.test.js

```diff
@@ -180,4 +183,31 @@ describe('elements/content-sidebar/ActiveState/activity-feed/ActiveState', () =>
             ).toBe(isCurrentVersion);
         },
     );
+
+    test("should have ActivityItem focused when it's activeFeedEntry", () => {
+        const wrapper = getShallowWrapper({
+            activeFeedItem: comment,
+        }).dive();
+
+        expect(getComment(wrapper).prop('isFocused')).toBe(true);
+    });
+    
+    test.each`
+      isSelected | expectedValue
+      ${false} | ${null}
+      ${true} | ${comment.id}
```

Comment:
The spacing of this file seems off. Please avoid pushing with `--no-verify`, as it skips important linting and formatting steps.